docker/sawtooth-dev-python

```diff
@@ -86,8 +86,9 @@ RUN echo "deb http://repo.sawtooth.me/ubuntu/ci xenial universe" >> /etc/apt/sou
  && pip3 install \
     pylint \
     bandit \
-    pyformance \
-    coverage --upgrade
+    coverage --upgrade \
+&& pip3  install git+https://github.com/omergertel/pyformance.git \
```

Comment:
We will need to package this as a deb package since this is a runtime dependency.